pkgs/applications/networking/cluster/terranix/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, ... }:
+
+stdenv.mkDerivation {
+
+  pname = "terranix";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "mrVanDalo";
+    repo = "terranix";
+    rev = "2.3.0";
```

Maybe reuse `version`?

```suggestion
    rev = version;
```